pyproject.toml

```
@@ -38,6 +39,7 @@ Issues = "https://github.com/CMIP-Data-Request/CMIP7_DReq_Software/issues"
 # modules and functions that act as command line utilities to be specified here
 # use pip install -e . from the root directory of the repository to set up the entry points
 export_dreq_lists_json = "data_request_api.command_line.export_dreq_lists_json:main"
+CMIP7_data_request_api_config = "data_request_api.command_line.config:main"
 
 [tool.pytest.ini_options]
 cache_dir = "tests/.pytest_cache"
```

I think `cache_dir` on line 45 needs a quick update as the `tests` directory no longer exists.

```suggestion
cache_dir = "./.pytest_cache"
```